src/main/java/uk/gov/hmcts/reform/notificationservice/task/NotificationMessageProcessTask.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.notificationservice.task;
+
+import com.microsoft.azure.servicebus.primitives.ServiceBusException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.notificationservice.service.NotificationMessageProcessor;
+
+@Service
+@ConditionalOnProperty(value = "scheduling.task.notifications-consume.enabled", matchIfMissing = true)
+public class NotificationMessageProcessTask {
+
+    private static final Logger log = LoggerFactory.getLogger(NotificationMessageProcessTask.class);
+
+    private final NotificationMessageProcessor notificationMessageProcessor;
+    private static final String TASK_NAME = "notification consume task";
```

Comment:
Still wrong task name ;)

```suggestion
    private static final String TASK_NAME = "consume-notifications";
```

this ^ is just a suggestion but wording sounds good to me :man_shrugging: